website/docs/d/batch_pool.html.markdown

```diff
@@ -110,3 +112,19 @@ A `certificate` block exports the following:
 -> **NOTE:** This property is applicable only for pools configured with Windows nodes (that is, created with cloudServiceConfiguration, or with virtualMachineConfiguration using a Windows image reference).
 
 * `visibility` - Which user accounts on the compute node have access to the private data of the certificate.
+
+---
+
+A `resource_file` block exports the following:
+
+* `auto_storage_container_name` - The storage container name in the auto storage account.
+
+* `blob_prefix` - The blob prefix used when downloading blobs from an Azure Storage container.
+
+* `file_Mode` - The file permission mode attribute in octal format. 
```

Comment:
I think it's worth calling out this is a string here, since octal format could be interpreted as an int too:

```suggestion
* `file_mode` - The file permission mode attribute represented as a string in octal format (e.g. `"0644"`).
```